src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Options/EditorConfig/EditorConfigStorageLocation.cs

```diff
@@ -12,9 +12,6 @@ internal static class EditorConfigStorageLocation
         public static EditorConfigStorageLocation<bool> ForBoolOption(string keyName)
             => new(keyName, s_parseBool, s_getBoolEditorConfigStringForValue);
 
-        public static EditorConfigStorageLocation<int> ForInt32Option(string keyName)
-            => new(keyName, s_parseInt32, s_getInt32EditorConfigStringForValue);
-
         public static EditorConfigStorageLocation<string> ForStringOption(string keyName, string emptyStringRepresentation)
             => new(keyName, s_parseString, (string value) => string.IsNullOrEmpty(value) ? emptyStringRepresentation : s_getStringEditorConfigStringForValue(value));
```

Comment:
`s_parseBool` should come from the editorconfig data object and the function removed. Same with `GetBoolEditorConfigStringForValue` and `s_getBoolEditorConfigStringForValue` and `ParseString` and `s_parseString`